### ISSUE TITLE
Update the ReadMe with the encoding standard used 

### ISSUE THREAD

**Issue by tristancr** [jwerle/b64.c#30]
Hi, 

I found your code really good. It seems that the current implementation follows the RFC 4648 standard. 

Could you please confirm that it is the case ? If so, could you update the ReadMe so it makes clear which standard you are implementing to everyone ? 

Thanks !

**Comment by jwerle** [comment]
Yep that is the standard we based this on. I'd happily accept a PR as I am pretty busy with other things at the moment. I'l leave this issue open as reminder in case I can get to it sometime soon 🤞🏼
